convert2rhel/initialize.py

```diff
@@ -49,6 +63,9 @@ def run():
     # prepare environment
     set_locale()
 
+    # Initialize logging to stop duplicate messages.
+    initialize_logging()
```

Comment:
I think not.  The reason being that we should try to keep the early initialization just for things that really need to be initialized early.  Otherwise, down the road, we might find out that there is something that needs to be initialized before `main.initalize_logger()` and then we'd have to figure out how to do that.  I will rename the function though since you have a good point that the names are very similar so it is confusing why we have two things that initialize logging.

Reply:
Nice, good point. At first, I thought that this function was the one from main, but then realized that they are almost the same.